### PR TITLE
Switch to final round for PEGS

### PR DESCRIPTION
**Pull request by jaymedina**
> [!CAUTION]
> Not to be merged until the final round begins on August 6th

1. This PR updates the PEGS dag switching it to the final round profile on `nf-synapse-challenge`: https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/blob/0b45b1de981eb3e0c854bbbedbd6a570d95dbde2/nextflow.config#L42
2. This also points to the latest `nf-synapse-challenge` with [these changes](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/pull/39)

## testing

Successful [workflow run](https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/pegs-challenge-project/watch/51AVoBPkjLbHcx) on test submission:

image

### CODE REVIEW

File: dags/pegs-challenge-submission-dag.py [hunk 1]
```diff
@@ -20,9 +20,9 @@
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
     "aws_conn_id": Param("AWS_TOWER_PROD_S3_CONN", type="string"),
     "revision": Param("0b45b1de981eb3e0c854bbbedbd6a570d95dbde2", type="string"),
-    "challenge_profile": Param("pegs_challenge_validate", type="string"),
+    "challenge_profile": Param("pegs_challenge_test", type="string"),
     "tower_conn_id": Param("PEGS_CHALLENGE_PROJECT_TOWER_CONN", type="string"),
-    "tower_view_id": Param("syn57373526", type="string"),
+    "tower_view_id": Param("syn52817032", type="string"),
```
Comment (vpchung) [inline dags/pegs-challenge-submission-dag.py]:
```suggestion
    "tower_view_id": Param("syn58942525", type="string"),
```

syn52817032 took me to the challenge website, but if we want the Submission View, then syn58942525 is the one to use.

**Review by vpchung — CHANGES_REQUESTED** [review]

File: dags/pegs-challenge-submission-dag.py (hunk 1 above)
Comment (jaymedina) [reply dags/pegs-challenge-submission-dag.py]: Thanks for double-checking!

**Comment by vpchung** [comment]
Can you remind me again, does the "test" profile have the option to not return the scores in the email?

**Comment by jaymedina** [comment]
The `test` profile _does_ return the scores. We'll have to update it so that `send_email` param is set to `false` before merging this, if we don't want that. I'll make another PR over there and update this one after merge.

**Review by vpchung — APPROVED** [review]

**Review by BryanFauble — APPROVED** [review]

**Pull request merged**